verilog/tools/kythe/BUILD

```diff
@@ -0,0 +1,95 @@
+# 'verible-verilog-kythe-extractor' is a program for extracting Verilog/SystemVerilog to kythe facts.
+
+licenses(["notice"])
+
+# Generate foreach list for IndexingFactType enum.
```

Comment:
It's preferred to write comments in format:
"Generate**s** foreach..."
"Returns Token..."
"Deletes ..."